code/game/machinery/doors/airlock.dm

```diff
@@ -1465,10 +1465,21 @@ About the new airlock wires panel:
 			cutting = TRUE
 	else if(istype(tool, /obj/item/crowbar/robotic/jawsoflife))
 		if(bolt_cut_state == BOLTS_FINE)
-			to_chat(user, SPAN_WARNING("You force the bolt cover open!"))
-			playsound(src, 'sound/weapons/smash.ogg', 100, extrarange = SHORT_RANGE_SOUND_EXTRARANGE)
-			bolt_cut_state = BOLTS_EXPOSED
+			user.visible_message(\
```

Comment:
you don't need to make newlines like this, put the first span_danger in the same line and align the others below, and it should work